paper-menu-button.html

```diff
@@ -79,28 +79,31 @@
         @apply(--paper-menu-button-disabled);
       }
 
-      :host([vertical-align="top"]) paper-material {
-        margin-bottom: 20px;
-        margin-top: -10px;
-        top: 10px;
-      }
-
-      :host([vertical-align="bottom"]) paper-material {
-        bottom: 10px;
-        margin-bottom: -10px;
-        margin-top: 20px;
-      }
-
       iron-dropdown {
         @apply(--paper-menu-button-dropdown);
       }
 
       .dropdown-content {
+        @apply(--shadow-elevation-2dp);
```

Comment:
I think that this is bad news bears, because the user could've been using this class for styling the inner menu, and you're using this class to style the outer material.